synapse/lib/link.py

```diff
@@ -68,6 +68,12 @@
 
     return link0, file1
 
+async def linksock():
```

Comment:
This will be used to facilitate comms with tasks running outside the asyncio loop ( possibly in a sub-process ).  There is one use case in an extension / private-module already.

Reply:
Can we add a test in test_lib_link for it then?